artiq/frontend/artiq_master.py

```diff
@@ -112,7 +120,8 @@ def ccb_issue(service, *args, **kwargs):
         "master_device_db": device_db,
         "master_dataset_db": dataset_db,
         "master_schedule": scheduler,
-        "master_experiment_db": experiment_db
+        "master_experiment_db": experiment_db,
+        "master_config": config
```

Comment:
Trivial style issue, but I think it should be first in that list, since this is the first thing that gets accessed.